ert_shared/ensemble_evaluator/ensemble/builder.py

```diff
@@ -704,6 +708,7 @@ def build(self):
                 jobs,
                 stage.get_name(),
                 source,
+                run_path=self._run_path,
```

Comment:
Can we avoid forwarding the runpath to the function step, since you should not assume that you have a file system available when running it?

Reply:
(?)  The run-path is only forwared to `_UnixStep`, not to `_FunctionStep`.

Reply:
Ahh, Github confused me in the way it showed the diff, my bad!